dapp/src/components/CommitHistory.astro

```diff
@@ -0,0 +1,41 @@
+---
+import { getCommitHistory } from '../service/GithubService';
+import CommitRecord from './CommitRecord.astro';
+import { formatDate } from '../service/utils';
+import type { FormattedCommit } from '../types/github';
+
+const commitHistory = await getCommitHistory('tupui', 'soroban-versioning');
```

Comment:
I am not sure I understand why this is set to `'tupui', 'soroban-versioning'`. We want to fetch the history of the project people would select, not of `tupui/soroban-versioning`. This is missing a props or something else so that it points to the GitHub of the selected project (url in the config like in the Project info section).

Reply:
yes, you're right
that's why I said I used demo data
I've developed function to get dynamic info user selects and testing now
after test, I'll push it too